exp/constraints.lisp

```diff
@@ -358,29 +358,22 @@
      (* (ebs_cutoff) (- 1 (comp)))))
 
 ;; 2
-(defconstraint preprocessing-2-modexp-log (:perspective macro :guard IS_MODEXP_LOG)
+(defconstraint preprocessing-2-modexp-log-min-cutoff-less-or-equal-than-16 (:perspective macro :guard IS_MODEXP_LOG)
   (callToLT 2 0 (min_cutoff) 0 17))
 
 (defun (min_cutoff_leq_16)
   (shift preprocessing/WCP_RES 2))
 
 ;; 3
-(defconstraint preprocessing-3-modexp-log (:perspective macro :guard IS_MODEXP_LOG)
-  (callToLT 3 0 (ebs_cutoff) 0 17))
-
-(defun (ebs_cutoff_leq_16)
-  (shift preprocessing/WCP_RES 3))
-
-;; 4
-(defconstraint preprocessing-4-modexp-log (:perspective macro :guard IS_MODEXP_LOG)
-  (callToISZERO 4 0 (raw_lead_hi)))
+(defconstraint preprocessing-4-modexp-log-raw-lead-hi-is-zero (:perspective macro :guard IS_MODEXP_LOG)
+  (callToISZERO 3 0 (raw_lead_hi)))
```

Comment:
If you keep the number in there e.g. the `4` in `preprocessing-4-modexp-log-raw-lead-hi-is-zero` you are creating issues for moments such as this. Please rename these and similar contraints for `exp` without the numbers and using the following convention:
```lisp
(defconstraint    preprocessing---modexp---raw-lead-hi-is-zero ...)
```
And similarly for other constraints.